classes/Hook/HookDisplayHeader.php

```diff
@@ -120,14 +118,6 @@ private function displayGaTag()
      */
     public function setBackOffice($backOffice)
     {
-        $this->acknowledgeBackOfficeContext($backOffice);
-    }
-
-    /**
-     * @param bool $isBackOffice
-     */
-    public function acknowledgeBackOfficeContext($isBackOffice)
-    {
-        $this->backOffice = $isBackOffice;
+        $this->backOffice = $backOffice;
```

Comment:
@matks disagreed with [this kind of change last year](https://github.com/PrestaShop/ps_googleanalytics/pull/122/files). Is it OK this time ? :stuck_out_tongue_winking_eye:

Reply:
This version of module will break lot of things, so we can finally remove this nonsense. :-))

Reply:
@leemyongpakvn In the PR https://github.com/PrestaShop/ps_googleanalytics/pull/122 the BC break was alone, so merging it would have required to release a major version just for one little BC break. I felt like it was not a good balance.

If we aim for a bigger major release, packed with BC breaks, I think we can accept it.

It's like going to the supermarket with your car 😄 if you tell me "I just want to buy a bottle of water" I feel like it's not worth it (using the car, using gas, spending time driving... all this just for one bottle) ; if you tell me "I'm doing the groceries for the two next weeks" then it makes sense